src/Models/IoTContainerizedProject.ts

```diff
@@ -76,7 +75,7 @@ export class IoTContainerizedProject extends IoTWorkbenchProjectBase {
     openInNewWindow: boolean
   ): Promise<void> {
     // Can only create project locally
-    await RemoteExtension.checkRemoteExtension();
+    await RemoteExtension.checkRemoteExtension("create iot containerized project");

```

Comment:
The same operation string appears in many places.

Reply:
See above.